paddlenlp/transformers/bert/modeling.py

```diff
@@ -508,6 +509,41 @@ def forward(self,
         logits = self.classifier(sequence_output)
         return logits
 
+class BertForMultiLabelTextClassification(BertPretrainedModel):
+    """
+    Model for multi label text classification task with BERT.
+    Args:
+        bert (BertModel): An instance of BertModel.
```

Comment:
bert (BertModel, nn.Layer): An instance of BertModel.

Reply:
done